test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh

```diff
@@ -4,23 +4,21 @@
 
 set -e
 
+readonly DEFAULT_VALIDITY_DAYS=${DEFAULT_VALIDITY_DAYS:-730}
```

Comment:
the permission change of this file is not desired? (changes from `755` to `644`)

Reply:
Oh yeah, probably an artifact of developing on windows, will fix